zero-thrs-control/src/thrs/input_output/modules/pvt.py

```diff
@@ -273,6 +282,96 @@ class PvtSensorValues(ThrsModel):
         sensor.TemperatureSensor, component_meta(yard_tag="50009005-45")
     ]
 
+    @computed_field(
+        json_schema_extra=component_meta(included_in_fmu=False).json_schema_extra
+    )
+    @property
+    def pvt_max_temperature_main_fwd_strings(
+        self,
+    ) -> Annotated[sensor.CalculatedTemperature, component_meta(included_in_fmu=False)]:
+        max_sensor = max(
+            [
+                self.pvt_temperature_main_string_1_1_return,
+                self.pvt_temperature_main_string_1_2_return,
+                self.pvt_temperature_main_string_2_1_return,
+                self.pvt_temperature_main_string_2_2_return,
+                self.pvt_temperature_main_string_3_return,
+                self.pvt_temperature_main_string_4_return,
+                self.pvt_temperature_main_string_5_1_return,
+                self.pvt_temperature_main_string_5_2_return,
+                self.pvt_temperature_main_string_6_1_return,
+                self.pvt_temperature_main_string_6_2_return,
+                self.pvt_temperature_main_string_1_supply,
+                self.pvt_temperature_main_string_2_supply,
+                self.pvt_temperature_main_string_3_supply,
+                self.pvt_temperature_main_string_4_supply,
+                self.pvt_temperature_main_string_5_supply,
+                self.pvt_temperature_main_string_6_supply,
+            ],
+            key=lambda sensor: sensor.temperature.value,
+        )
+
+        return sensor.CalculatedTemperature(**max_sensor.model_dump())
+
+    @computed_field(
+        json_schema_extra=component_meta(included_in_fmu=False).json_schema_extra
+    )
+    @property
+    def pvt_max_temperature_main_aft_strings(
+        self,
+    ) -> Annotated[sensor.CalculatedTemperature, component_meta(included_in_fmu=False)]:
+        max_sensor = max(
+            [
+                self.pvt_temperature_main_string_7_1_return,
+                self.pvt_temperature_main_string_7_2_return,
+                self.pvt_temperature_main_string_8_1_return,
+                self.pvt_temperature_main_string_8_2_return,
+                self.pvt_temperature_main_string_9_return,
+                self.pvt_temperature_main_string_10_return,
+                self.pvt_temperature_main_string_11_1_return,
+                self.pvt_temperature_main_string_11_2_return,
+                self.pvt_temperature_main_string_12_return,
+                self.pvt_temperature_main_string_13_return,
+                self.pvt_temperature_main_string_7_supply,
+                self.pvt_temperature_main_string_8_supply,
+                self.pvt_temperature_main_string_9_supply,
+                self.pvt_temperature_main_string_10_supply,
+                self.pvt_temperature_main_string_11_supply,
+                self.pvt_temperature_main_string_12_supply,
+                self.pvt_temperature_main_string_13_supply,
+            ],
+            key=lambda sensor: sensor.temperature.value,
+        )
```

Comment:
Do you think it's worthwhile to deduplicate this pattern into a `max_temp(sensors: list[TemperatureSensor]) -> CalculatedTemperature`?